apps/finance/contracts/Finance.sol

```diff
@@ -271,6 +271,30 @@ contract Finance is App, Initializable, ERC677Receiver {
         ChangePaymentState(_paymentId, _disabled);
     }
 
+    /**
+     * @dev Allows make a simple payment from this contract to Vault,
+            to avoid locked tokens in contract forever.
+            This contract should never receive tokens with a simple transfer call,
+            but in case it happens, this function allows to recover them.
+     * @notice Send tokens to Vault
+     * @param _token Token whose balance is going to be transferred.
+     * @return success boolean indicating whether transaction suceeded
+     */
+    function depositToVault(address _token) public returns (bool success) {
+        ERC20 token = ERC20(_token);
+        uint256 value = token.balanceOf(this);
+        require(value >= 0);
+
+        _recordIncomingTransaction(
+            token,
+            this,
+            value,
+            "Transfer from Finance"
+        );
+        require(token.transfer(address(vault), value));
+        return true;
```

Comment:
Unsure why the return boolean is needed, as it always will either fail through a revert or return true. 

I propose to either remove the return value altogether or do `return token.transfer(address(vault), value)`, I think both would be fine, but I would side with removing the return value tbh.